### ISSUE TITLE
[Advise] Tell me if I'm doing this right

### ISSUE THREAD

**Issue by TomerGamerTV** [Cubitect/cubiomes-viewer#272]
I'm trying to search for a specific seed but I'm not sure if I'm doing it right, I tried playing with the settings and from 170m seeds per second it dropped to 1-2... 
![image](https://github.com/Cubitect/cubiomes-viewer/assets/44266242/24898ba8-c61f-4ba4-b424-71b1a6e416df)

Can somebody help me?
Also 48bit family didn't change the speed of the scan

**Comment by black-backdoor** [comment]
Would you mind describing the problem in more detail?

I tried searching for such seeds with the same settings:

![image](https://github.com/Cubitect/cubiomes-viewer/assets/94786430/461b6596-e4f1-486c-b795-496ed121cfa3)


here are my settings when looking for results:
[settings.txt](https://github.com/Cubitect/cubiomes-viewer/files/13548549/settings.txt)

**Comment by TomerGamerTV** [comment]
There is no problem, i just wenna know if I'm searching right or doing something wrong so I won't waste time trying to find something that cannot be found.

**Comment by black-backdoor** [comment]
Check you biomes settings everything else looks fine

sorry, the last message wasn't on purpose

**Comment by Boobies** [comment]
Should probably take this elsewhere. Bug trackers are for bugs and this behavior violates GitHub's community code of conduct.

**Comment by TomerGamerTV** [comment]
> Should probably take this elsewhere. Bug trackers are for bugs and this behavior violates GitHub's community code of conduct.

take where? it's better that I asked here first and didn't make an issue about a bug that didn't exist. Also this is not a bug tracker its an issue and I've seen worse on github and nothing happened to them.

**Comment by Boobies** [comment]
Yes, issues regarding the project rather than people's personal issues. Also, nothing happening isn't an excuse for misbehaving. Plenty of people litter and nothing happens to them; that doesn't mean it's nice to throw your trash in the middle of the street. Or should we only avoid misbehaving when there are consequences?

**Comment by black-backdoor** [comment]
Stop arguing about where the Question / Advice belongs to.

I have some questions for you guys:

1.  Can we go back to solving the problem / answering the question?
2.  Can we create an environment where people can go to get help and advise? Like a wiki.

**Comment by TomerGamerTV** [comment]
> Stop arguing about where the Question / Advice belongs to.
> 
> I have some questions for you guys:
> 
> 1. Can we go back to solving the problem / answering the question
> 2. Can we create an environment where people can go to get help and advise, Like a wiki.

Making a wiki is a great idea but sadly we can't talk to a wiki, a subreddit would be nice.

**Comment by black-backdoor** [comment]
> > Stop arguing about where the Question / Advice belongs to.
> > I have some questions for you guys:
> > 
> > 1. Can we go back to solving the problem / answering the question
> > 2. Can we create an environment where people can go to get help and advise, Like a wiki.
> 
> Making a wiki is a great idea but sadly we can't talk to a wiki, a subreddit would be nice.

I maybe also a discord?
